### PR TITLE
[PM-1699] Updated low kdf iterations warning

### PR DESCRIPTION
**Pull request by gbubemismith**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [X] Other
```

## Objective
Update the low KDF warning card.

## Code changes

Changed the card body massage from `Increase your KDF encryption settings to improve the security of your account.` to `Update your encryption settings to meet new security recommendations and improve account protection.` and the link message to `Update KDF settings`.

## Screenshots

https://user-images.githubusercontent.com/13024008/230195570-12535b61-a0a6-4d30-af23-de4bedc39af3.mov

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: apps/web/src/locales/en/messages.json [hunk 1]
```diff
@@ -6497,7 +6497,7 @@
     "message": "Low KDF Iterations"
   },
   "lowKdfIterationsDesc": {
```
Comment (djsmith85) [inline apps/web/src/locales/en/messages.json:6499]: Please also change the translation-key as your change in the source-string won't change the existing translations. Changing the key deletes all existing translations and all translators will get notified about a new string to translate.

File: apps/web/src/locales/en/messages.json [hunk 2]
```diff
@@ -6497,7 +6497,7 @@
...
-    "message": "Increase your KDF encryption settings to improve the security of your account."
+    "message": "Update your encryption settings to meet new security recommendations and improve account protection."
   },
   "changeKdfSettings": {
     "message": "Change KDF settings"
```
Comment (djsmith85) [inline apps/web/src/locales/en/messages.json:6503]: This should be removed as it's only occurrence is the place that got change with `updateKdfSettings`

**Review by djsmith85 — CHANGES_REQUESTED** [review]

**Review by djsmith85 — APPROVED** [review]
@gbubemismith Thanks, this is looking good now.

**Review by differsthecat — DISMISSED** [review]

**Comment by differsthecat** [comment]
@gbubemismith This looks good, looks like there are some conflicts that need to be resolved

**Comment by eliykat** [comment]
AC Team's review is requested here, but the only changes to our files are imports that are out of order. Note that linting is failing.

I think you need to run linting & prettier locally, then push changes. That should fix the job and drop off the extra teams from the reviewers.

**Comment by gbubemismith** [comment]
> AC Team's review is requested here, but the only changes to our files are imports that are out of order. Note that linting is failing.
> 
> I think you need to run linting & prettier locally, then push changes. That should fix the job and drop off the extra teams from the reviewers.

I think my ide was messing the files up a bit when I was merging. Everything looks good now.

**Pull request merged**